src/lib.rs

```diff
@@ -77,6 +77,16 @@ impl Builder {
         self
     }
 
+    pub fn whitelisted_function<T: Into<String>>(mut self, arg: T) -> Builder {
```

Comment:
Can you use `Borrow<str>` instead, to match with the signature of the `insert` function, and prevent an extra allocation in case you give it a `str`, for example?

r=me with that.

Thanks!